packages/create-remix/rollup.config.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Maybe we should disable this globally in our root `.eslintrc` file for these files?